store/postgres/instance_repository_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !unit_test
```

Comment:
yes, it is because of the go1.17. i can remove it because it is only a preferred syntax in 1.17 (i havent removed the old syntax either, kinda missed this). 

let me change it back so we stick to the one that compatible with 1.16

Reply:
removed these